src/dbnode/network/server/httpjson/handlers.go

```diff
@@ -29,16 +29,16 @@ import (
 	"reflect"
 	"strings"
 
+	"github.com/m3db/m3/src/query/api/v1/handler/prometheus/handleroptions"
```

Comment:
nit: Doesn't seem like a good path for this since it's not a prometheus (or even query) header

Reply:
True, will move these headers to a headers package.